### PR TITLE
IDEX-3134: Port setting breakpoints

### PR DESCRIPTION
**Pull request by tolusha**

### CODE REVIEW

File: plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+import org.eclipse.che.ide.editor.orion.client.jso.OrionAnnotationOverlay;
+import org.eclipse.che.ide.editor.orion.client.jso.OrionEditorOverlay;
+import org.eclipse.che.ide.editor.orion.client.jso.OrionExtRulerOverlay;
+import org.eclipse.che.ide.editor.orion.client.jso.OrionTextModelOverlay;
+import org.eclipse.che.ide.jseditor.client.gutter.Gutter;
+import org.eclipse.che.ide.jseditor.client.gutter.Gutters;
+import org.eclipse.che.ide.util.dom.Elements;
+
+/**
+ * @author Anatoliy Bazko
```
Comment (skabashnyuk) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java]: Need meaningful javadoc here.

File: plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionExtRulerOverlay.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.editor.orion.client.jso;
+
+import com.google.gwt.core.client.JavaScriptObject;
+
```
Comment (skabashnyuk) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionExtRulerOverlay.java]: Need meaningful javadoc here.

File: plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java (hunk 1 above)
Comment (tolusha) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java]: Gutter interface contains all necessary javadocs

Comment (skabashnyuk) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java]: I mean we need documentation for class

Comment (tolusha) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java]: Why  {@inheritDoc} annotation doesn't count?

**Comment by vparfonov** [comment]
ok

File: plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionExtRulerOverlay.java (hunk 2 above)
Comment (tolusha) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/jso/OrionExtRulerOverlay.java]: Added

File: plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java (hunk 1 above)
Comment (tolusha) [inline plugin-orion/che-plugin-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionBreakpointRuler.java]: Added

**Pull request merged**